examples/angular/src/app/guards/can-match-page.guard.ts

```diff
@@ -27,7 +27,7 @@ export const canMatchPage: CanMatchFn = async (
 
     const { vanityUrl } = entity;
 
-    if (vanityUrl?.permanentRedirect) {
+    if (vanityUrl?.permanentRedirect || vanityUrl?.temporaryRedirect) {
       router.navigate([vanityUrl.forwardTo]);
```

Comment:
@KevinDavilaDotCMS For redirects, return an Url Tree like this: `return router.createUrlTree([vanityUrl.forwardTo]);`

Reply:
Changed!